client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -134,13 +135,14 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                         {investigatorNameMsg} :
                                                     <b>
                                                             {option.value.userName}
-                                                        </b>
+                                                    </b>
+                                                        &nbsp;&nbsp;
```

Comment:
Is this necessary?

Reply:
yes, i wanted to have some space between the investigator's name and the sourceOrganization/desk